### PR TITLE
Add file

### PR DESCRIPTION
**Pull request by hapo-chidl**
hello hiếu

### CODE REVIEW

**Comment by hapo-hieupm** [comment]
Conflicting files

**Review by hapo-hieupm — COMMENTED** [review]
conflict

**Comment by hapo-chidl** [comment]
> conflict



> conflict

:))

File: test.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
 20200713
-ax
+
+
+=======
+
+branch feature/test-pr
+master
```
Comment (hapo-hieupm) [inline test.txt:7]: ok

**Pull request merged**